src/ui/components/trinary_input_char.c

```diff
@@ -61,20 +61,39 @@ typedef struct {
 } data_t;
 
 static void _set_alphabet(component_t* component, const char* alphabet)
+{
+    data_t* data = (data_t*)component->data;
```

Comment:
remove this function? it is too small to be worth it ;)

Reply:
Yeah will do.